crypto-ffi/bindings/swift/WireCoreCryptoUniffi/WireCoreCryptoUniffi/core_crypto_ffiFFI.h

```diff
@@ -0,0 +1,1943 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
+// Trust me, you don't want to mess with it!
```

Comment:
If this is autogenerated, why include it in the repo?

Reply:
Ah right forget to update the .gitignore